exercises/practice/triangle/.meta/Triangle.example.ps1

```diff
@@ -0,0 +1,46 @@
+Enum Triangle {
```

Comment:
I think that everything looks good.

But I think using an enum is a really good approach, but that may not be so obvious for the user that should be implemented.

I see two options to this:

1. Give the student the enum as stub, the negative part is that the user will not learn how to define it.
2. Make appended instructions there we say that an enum should be implemented, I can do this for you if you like, otherwise this `append` should be in a file called "instructions.append.md".

Reply:
I chose option 2 and give a quick introduction about Enum with an example. I think that should be enough hint, however if you think there should be more hint you can throw in a comment line saying it should be implemented.